pkg/utils/gateway_wrapper.go

```diff
@@ -0,0 +1,130 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+
+	gatewayapiv1 "sigs.k8s.io/gateway-api/apis/v1"
+)
+
+const (
+	SingleClusterAddressValue = "kudarant.io/single"
```

Comment:
This isn't an address value, `SingleClusterNameValue` might be more appropriate.